src/cli/utils.py

```
@@ -51,16 +51,29 @@ def get_store(config: CLIConfig) -> Any:
-    if config.embedder_type == "fastembed":
+    embedder_type = embedder_override or config.embedder_type
+
+    if embedder_type == "openai":
+        try:
+            from src.chunker import OpenAIEmbedder
+
+            return OpenAIEmbedder(model_name=config.model_name)
+        except ImportError:
+            raise ImportError(
+                "OpenAI is not installed. Run: pip install openai"
+            )
```

Ruff (0.14.10)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)